pkg/net/throughput.go

```diff
@@ -0,0 +1,59 @@
+package net
+
+import (
+	"sync"
+)
+
+var transmitLock sync.Mutex
+var receiveLock sync.Mutex
+
+// Throughput holds information about the amount of bytes
+// received and sent by a minio node, both historically and currently.
+type Throughput struct {
+	TotalReceived    uint64 `json:"totalReceived"`
+	TotalTransmitted uint64 `json:"totalTransmitted"`
+	Receiving        uint64 `json:"receiving"`
+	Transmitting     uint64 `json:"transmitting"`
+}
```

Comment:
The things being computed in this file are just counters of received/send/receiving/sending bytes - this is not called a "throughput". A throughput is a rate of something. To compute a throughput it is needed to divide by the time taken for each transfer and then maybe compute an average. Otherwise this should just be renamed as network counters or similar.

Reply:
Isn't using `sync.Atomic` an option here? using sync.Mutex + (counter++) better is to use sync.Atomic

Reply:
Then why don't we include that in the API too? Calling this is a throughput is an exaggeration.

Reply:
So, `NetStats`?

Reply:
> So, `NetStats`?

Yes definitely!